src/components/input/Select.tsx

```diff
@@ -405,6 +433,19 @@ type BaseSelectProps = CompositeProps & {
 
   /** A callback passed to the listbox onScroll */
   onListboxScroll?: JSX.HTMLAttributes<HTMLUListElement>['onScroll'];
+
+  /**
+   * Indicates how overflowing content should be handled in the listbox.
+   * * `truncate` -> Truncate the options so that they all fit in one line. An
```

Comment:
```suggestion
   * Indicates how overflowing content should be handled in the listbox.
   *
   * * `truncate` -> Truncate the options so that they all fit in one line. An
```

There should be a blank line between the one-line summary and detailed description.